data-visualization/data-visualization.md

```diff
@@ -211,7 +211,7 @@ const g = d3.select("svg")
 - [Data Visualization – How to Pick the Right Chart Type?](https://eazybi.com/blog/data_visualization_and_chart_types/)
 - [Data Visualization 101: How to Choose the Right Chart or Graph for Your Data](https://blog.hubspot.com/marketing/types-of-graphs-for-data-visualization)
 
-#### For inspiration and general related reading
+#### For inspiration and generally related reading
```

Comment:
maybe just take out the word 'generally', it doesnt help the title much.

Reply:
@alodahl ma'am done